.env.example

```diff
@@ -66,7 +65,8 @@ SERVER_ID='1'
 SLACK_CLIENT_ID='key_SLACK_CLIENT_ID'
 SLACK_CLIENT_SECRET='key_SLACK_CLIENT_SECRET'
 SOCKET_PORT='3001'
-STRIPE_SECRET_KEY='key_STRIPE_SECRET_KEY'
-STRIPE_PUBLISHABLE_KEY='key_STRIPE_PUBLISHABLE_KEY'
+# https://stripe.com/docs/keys
+STRIPE_SECRET_KEY='sk_test_4eC39HqLyjWDarjtT1zdp7dc'
```

Comment:
Use the Stripe keys from doc examples so that the client does not immediately throw an error that the key is incorrect.